depobs/worker/tasks.py

```diff
@@ -48,6 +49,7 @@
 from depobs.util.type_util import Result
 from depobs.worker import k8s
 
+import json, requests
```

Comment:
nit: usually order imports in groups of stdlib, third party, current app and we can drop the json import if we use `response.json()` https://requests.readthedocs.io/en/latest/user/quickstart/#json-response-content